### PR TITLE
More updates to our platform support page

### PR DESCRIPTION
**Pull request by tas50**
- Remove EOL Chef Automate Job Runners
- Add Oracle Linux to Chef Backend and Chef Manage
- Add Amazon Linux EOL policy
- Add RHEL 8 to Chef Backend

### CODE REVIEW

File: content/platforms.md [hunk 1]
```diff
@@ -386,13 +386,11 @@ for platforms that use:
...
 
-### Chef Automate Job Runners
+### Chef Backend
 
 #### Commercial Support
 
-Chef Automate Job Runners are supported on the Commercial Support
-platforms for Chef Automate Server listed above as well as on the
-following platforms:
+The following table lists the commercially-supported platforms for Chef Backend, the high-availability solution for Chef Infra Server:
```
Comment (IanMadd) [inline content/platforms.md]:
```suggestion
The following table lists the commercially-supported platforms for Chef Backend, which is the high-availability solution for Chef Infra Server:
```

Comment (tas50) [reply content/platforms.md]: Chef Backend is "the high availability solution"

Comment (IanMadd) [reply content/platforms.md]: How about: 
The following table lists the commercially-supported platforms for Chef Backend, which is the high-availability solution for Chef Infra Server:

Comment (tas50) [reply content/platforms.md]: that's fine with me

**Review by IanMadd — APPROVED** [review]

**Pull request merged**